stock_picking_customer_ref/models/stock_picking.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 AvanzOSC - Alfredo de la Fuente
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    @api.depends('group_id', 'sale_id.client_order_ref')
+    def _compute_client_order_ref(self):
+        for picking in self:
+            if picking.group_id:
+                cond = [('procurement_group_id', '=', picking.group_id.id)]
+                sale = self.env['sale.order'].search(cond, limit=1)
+                picking.client_order_ref = sale.client_order_ref
```

Comment:
I think you need add condition to avoid error AttributeError: 'bool' object has no attribute 'client_order_ref' when variable sale is False.

Reply:
Thank you for the review, i'll update the code in the next push.

Reply:
requesting this change